### DISCUSSION TITLE
Awesome Grids CSS

### DISCUSSION THREAD

**Discussion by danilovaz** [frontendbr/forum#123] [Perguntas e Respostas]
Fale pessoal :smiley: 

Eu estou começando a estudar sobre Grids CSS (tanto o rascunho da W3C quanto as ferramentas que usam do conceito) e senti a falta de um lugar onde eu pudesse listar e encontrar facilmente os melhores tópicos sobre o mesmo (artigos, exemplos, ferramentas, slides de apresentações, etc), então resolvi criar esse repositório [Awesome Grids CSS](https://github.com/danilovaz/awesome-grids-css) e queria saber se acham uma boa?

Se sim, bora mandar uns PRs lá que tô a fim de descobrir coisas legais de Grids CSS :smile: 

E se não, por que? :stuck_out_tongue_closed_eyes:

**Comment by afonsopacifer** [comment]
Fala @danilovaz, achei massa vc querer reunir topicos sobre grids, muita gente tem essa duvida. Muito bom!

Agora sobre a a Spec da W3C,  quando comecei a estudar ela, eu juntei alguns links em um repo tbm, tem bastante coisa lá:
https://github.com/afonsopacifer/learning-css-grid-layout

: )

**Comment by danilovaz** [comment]
Que massa @afonsopacifer com certeza vou dar uma olhada com mais calma mais tarde.

É então eu comecei à estudar sobre e achei que um repo pra me organizar ajudaria, aliás foi o seu `awesome-flexbox` que me deu um embalo pra criar algo pra organizar os itens de `grids css` :smiley:

**Comment by diegokeiti** [comment]
@danilovaz Estava procurando algum artigo relacionado a um tempo.
Naice !

**Comment by danilovaz** [comment]
pô, legal @diegokeiti ! dá uma estrelinha :star: pra ajudar os irmãos :man: :stuck_out_tongue_winking_eye:  :joy:         e confere [esse repo aqui](https://github.com/afonsopacifer/learning-css-grid-layout) do @afonsopacifer  tem muita coisa sobre Grids
